src/tests/parser.rs

```diff
@@ -786,7 +786,11 @@ fn test_line_counter() {
         assert_eq!(line_offset, expected);
     }
 
-    let pipelines: Vec<_> = ps.ast.walk().filter_map(ast::JobPipeline::cast).collect();
```

Comment:
this commit should be squashed into the commit that removes Castable
(or precede that one),
so that every commit improves consistency